### ISSUE TITLE
Unable to load Transformers Shadows Rising using the latest Teknoparrot.dll

### ISSUE THREAD

**Issue by legend800** [teknogods/TeknoParrotUI#936]
Using the latest TP build from Oct 5 '24, I'm unable to get Transformers Shadows Rising to load. Screen goes black, then shows a part of my desktop(see pic below), then full black again, back and forth forever. If I press ESC, I can see my desktop again, but my keys/mouse don't do anything. The only way to recover input on my computer is to send to standby or restart it via case buttons.

I know the game takes a while to load but when it works correctly, you see the in-game loading text. That's not what's going on here. 

I've tried short/long game paths, 1080/2160p desktop resolutions and both shell and Transformer2 exes. I even tried wiped the profile and TP inis in the game folder but still same result. 

This seemed to start in late Aug/early Sep. An older teknoparrot x86 lib from early Aug works just fine. That was until Oct 4th where now that old dll doesn't fix the issue anymore and I have to use old build + old lib to load game. So this is possibly related to the pathing changes on Oct 4th for this game, but not sure.

**_Config: Windows 11, AMD 5900x CPU, Nvidia 3070_**

_Note: Another user in Slack also confirmed using the older Teknoparrot.dll (from Aug. 4th) help then start loading Sega Rally 3 again. Doesn't load using the latest Teknoparrot.dll. So seems like some lib changes don't play nice with certain configs out there._ 

<img width="963" alt="1" src="https://github.com/user-attachments/assets/7ebd44bf-afb6-47f2-a330-20868eaa28eb">

**Comment by Pete-McDonald** [comment]
Loads fine here with the most recent TP as of today 

![image](https://github.com/user-attachments/assets/50d3dcdd-acfc-47bd-b6d1-e7e31ae7a880)

i9 12000K
3080Ti

**Comment by legend800** [comment]
Yes, I should have mentioned it seems to be working fine for some. So it's some kind of compatibility thing. 

Worked great at launch but now for my config, it won't show the in-game loading screen. All my other teknoparrot.dll games, including Transformers 1, just fine.
